specification/portal/resource-manager/Microsoft.Portal/preview/2019-01-01-preview/examples/TenantConfiguration/CreateOrUpdateTenantConfiguration.json

```diff
@@ -0,0 +1,33 @@
+{
+  "parameters": {
+    "configurationName": "default",
+    "api-version": "2019-01-01-preview",
+    "tenantConfiguration": {
+      "properties": {
+        "enforcePrivateMarkdownStorage": true
+      }
+    }
+  },
+  "responses": {
+    "201": {
+      "body": {
+        "id": "/providers/Microsoft.Portal/tenantConfiguration/default",
```

Comment:
>default [](start = 63, length = 7)

Is default the only one that will be allowed or can you have more than one? (I don't think it makes sense to have more than one - just trying to understand how this works.)

Reply:
at this moment we are using singleton with the name "default" (it is defined as in swagger as by code validation), since we are creating this API for our purpose (the flag for dashboards), though we think that in theory this API may be used for other configurations